### ISSUE TITLE
installing project-origin-wallet package via Helm fails

### ISSUE THREAD

**Issue by InassBelgcir** [project-origin/wallet#54]
### Describe the Bug

When installing project-origin-wallet package, I get the following error : 

**Error: INSTALLATION FAILED: execution error at (project-origin-wallet/templates/deployment-wallet.yaml:34:24): A valid externalUrl is required!**

### To Reproduce

```bash
On a cloud shell : 

1.connect to the cluster
2.run helm repo add project-origin https://project-origin.github.io/helm-registry (successful)
3.run helm install my-project-origin-wallet project-origin/project-origin-wallet --version 0.1.2
```

### Expected Behavior

package installed

### Screenshots

![image](https://github.com/project-origin/wallet/assets/141126356/d02ea387-66dc-49b5-ac92-7b34fce317ae)

### Desktop

- OS: Windows10
- Browser: Microsoft Edge

**Comment by MartinSchmidt** [comment]
Hi @InassBelgcir,

The documentation is still not complete, but at a minimum you must configure `wallet.externalUrl` either in a values.yaml or `--set wallet.externalUrl=http://hello.com`

This is so the WalletSystem knows its own external url, this is needed as it is embedded into DepositEndpoints (previously known as WalletSections) so wallets can interact with each other.

I hope this helps.

**Comment by MartinSchmidt** [comment]
After a teams call, this solved the initial issue, new one came where the chart currently has an requirement that the postgres operater [CloudNativePG](https://cloudnative-pg.io) is installed, this should be fixed in issued #56 where MsSQL support will be included.
